code/modules/multiz/turf.dm

```diff
@@ -82,15 +82,15 @@
 	below.above = src
 	levelupdate()
 	for(var/atom/movable/A in src)
```

Comment:
Turfs can only contain movables, so this is probably safe to convert to a typeless for.